docs/windows.md

```diff
@@ -0,0 +1,210 @@
+  * If OVS driver is not signed, please refer to the Windows doc about how to [install a test-signed driver package on the test computer](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/installing-a-test-signed-driver-package-on-the-test-computer).
+  * If you don't have a self-signed OVS package and just want to try the Antrea on windows, Antrea provides a
+    test-signed OVS package for you. See details in [Join Windows worker Nodes](#Join-Windows-worker-nodes) section.
+
+### Installation
+#### Download & configure Antrea for Linux
+Configure the Antrea for Linux on master Node following [Getting started](/docs/getting-started.md) document.
+```
+# Example:
+kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/<TAG>/antrea.yml
+```
+#### Add Windows Antrea Agent and kube-proxy DaemonSets
+Now you can add Windows-compatible versions of Antrea Agent and kube-proxy by applying yaml files:
+- antrea-windows.yml
+  - Deploy Antrea Agent Windows DaemonSet.
+- kube-proxy-windows.yaml:
+  - Deploy kube-proxy Windows DaemonSet.
+
+Download and apply `antrea-windows.yml`.
+```
+# Example:
+kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/<TAG>/antrea-windows.yml
+```
+Download `kube-proxy.yaml` from kubernetes official repository and set kube-proxy version.
+```
+# Example:
+curl.exe -LO https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/kube-proxy.yml | sed 's/VERSION/v1.18.0/g' > kube-proxy.yaml
+```
+Replace the content of `run-script.ps1` in configmap named `kube-proxy-windows` as following:
+```
+apiVersion: v1
+data:
+  run-script.ps1: |-
+    $ErrorActionPreference = "Stop";
+    mkdir -force /host/var/lib/kube-proxy/var/run/secrets/kubernetes.io/serviceaccount
+    mkdir -force /host/k/kube-proxy
+
+    cp -force /k/kube-proxy/* /host/k/kube-proxy
+    cp -force /var/lib/kube-proxy/* /host/var/lib/kube-proxy
+    cp -force /var/run/secrets/kubernetes.io/serviceaccount/* /host/var/lib/kube-proxy/var/run/secrets/kubernetes.io/serviceaccount
+
+    wins cli process run --path /k/kube-proxy/kube-proxy.exe --args "--v=4 --config=/var/lib/kube-proxy/config.conf --proxy-mode=userspace --hostname-override=$env:NODE_NAME"
+
+kind: ConfigMap
+metadata:
+  labels:
+    app: kube-proxy
+  name: kube-proxy-windows
+  namespace: kube-system
+``` 
+Set the `hostNetwork` option as true in spec of kube-proxy-windows daemonset.
+```
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    k8s-app: kube-proxy
+  name: kube-proxy-windows
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      k8s-app: kube-proxy-windows
+  template:
+    metadata:
+      labels:
+        k8s-app: kube-proxy-windows
+    spec:
+      hostNetwork: true
+```
+Then apply the `kube-proxy.yaml`
+```
+kubectl apply -f kube-proxy.yaml
+```
+
+#### Join Windows worker Nodes
+1.(Optional, Test-Only) Install OVS provided by Antrea
```

Comment:
```suggestion
1. (Optional, Test-Only) Install OVS provided by Antrea
```
Need a space after "1." like "2." has, otherwise it doesn't appear as numbered list. So do others.

Reply:
Done